Dockerfile

```diff
@@ -1,39 +1,40 @@
 # Base
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS base
-WORKDIR /app
-ENV ASPNETCORE_ENVIRONMENT=production
+ARG REGISTRY=562955126301.dkr.ecr.eu-west-2.amazonaws.com
 
-# Development
-FROM base AS development
-WORKDIR /FFCDemoPaymentService
-RUN apt-get update \
-  && apt-get install -y --no-install-recommends unzip \
-  && curl -sSL https://aka.ms/getvsdbgsh | bash /dev/stdin -v latest -l /vsdbg
-COPY ./FFCDemoPaymentService/*.csproj ./
+# FROM $REGISTRY/ffc-dotnet-parent/sdk:$BASE_VERSION AS 
+FROM dotnet-sdk-parent as build
```

Comment:
Should this not use `dotnet-sdk-development` image?  Otherwise we're not using that at all

Reply:
i think the sdk should be treat as developement as the sdk image will never be used in prod so i have deleted the dotnet develoepemt folder